Sources/Atoms/Context/AtomTransactionContext.swift

```diff
@@ -1,7 +1,7 @@
-/// A context structure that to read, watch, and otherwise interacting with atoms.
+/// A context structure to read, watch, and otherwise interact with atoms.
 ///
-/// Through this context, watching of an atom is initiated, and when that atom is updated,
-/// the value of the atom to which this context is provided will be updated transitively.
+/// When an atom watching is initiated through this context, and that atom is updated,
```

Comment:
[Options]
1. When an atom watching is initiated through this context
2. When an atom is watched through this context
3. When watching an atom is initiated through this context

Reply:
I'd vote for 2!

Reply:
I'd vote for 2!